src/app/modules/showcase-ui/examples/table-example/table-example.component.ts

```diff
@@ -178,6 +178,10 @@ export class TableExampleComponent {
       cellTooltip: { text: (data) => data.nameLong, onlyOnOverflow: true },
       cellMaxWidth: '300px', // overrides the maxColumnWidth on the table
       columnOrder: 1,
+      click: (data, cell, mouseEvent) => {
+        mouseEvent.stopPropagation();
+        console.log('click cell content > function params:', { data }, { cell }, { mouseEvent });
```

Comment:
Remove log

Reply:
That's for the example and for debugging, I think its ok